packages/editor-ui/src/components/CommunityPackageInstallModal.vue

```diff
@@ -121,13 +118,15 @@ export default mixins(
 			} else {
 				try {
 					this.infoTextErrorMessage = '';
+					this.loading = true;
 					await this.$store.dispatch('communityNodes/installPackage', this.packageName);
+					await this.$store.dispatch('communityNodes/fetchInstalledPackages');
```

Comment:
would be nice to add a comment here to explain why this is needed...

Reply:
👍